### ISSUE TITLE
NewEng Branch Channlels Have Not Been Implmented

### ISSUE THREAD

**Issue by erievs** [cosmic-cat-yt/Cosmic-Cat#14]
### Type

Page Renderer

### Page URL

https://www.youtube.com/@BelkoWood/featured?feature=watch (or any channel)

### Branch

beta

### Version

0.8

### A brief description

Okay so for the hell of it I decided to see if this new branch would load on older browsers such as Chrome 70, and behold it worked! I also tested it on Firefox current, however channels do not work on this current moment, however this is a much better branch as the code is readable and supports older browsers.

### Screenshots or videos

![image](https://github.com/cosmic-cat-yt/Cosmic-Cat/assets/65189871/193635e7-8c15-4d7a-95de-7063b4b32ff9)

### Console output

VM170:23 [Deprecation] 'webkitURL' is deprecated. Please use 'URL' instead.
n @ VM170:23
featured?feature=watch:477 Uncaught SyntaxError: Unexpected token .
userscript.html?name=Cosmic%20Cat.user.js&id=e3aa1af9-22e3-4b2a-831e-17c7d4f83803:2529 Storage.init storage has init.
raw.githubusercontent.com/cosmic-cat-yt/cosmic-cat-i18n/main/version.json:1 Failed to load resource: the server responded with a status of 404 ()
userscript.html?name=Cosmic%20Cat.user.js&id=e3aa1af9-22e3-4b2a-831e-17c7d4f83803:6941 SyntaxError: Unexpected token : in JSON at position 3
    at fetch.then.a (userscript.html?name=Cosmic%20Cat.user.js&id=e3aa1af9-22e3-4b2a-831e-17c7d4f83803:6938)
fetch.then.then.catch.err @ userscript.html?name=Cosmic%20Cat.user.js&id=e3aa1af9-22e3-4b2a-831e-17c7d4f83803:6941
featured?feature=watch:1 Access to fetch at 'https://accounts.google.com/ServiceLogin?service=youtube&passive=1209600&continue=https%3A%2F%2Fwww.youtube.com%2FgetAccountSwitcherEndpoint&followup=https%3A%2F%2Fwww.youtube.com%2FgetAccountSwitcherEndpoint' (redirected from 'https://www.youtube.com/getAccountSwitcherEndpoint') from origin 'https://www.youtube.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
userscript.html?name=Cosmic%20Cat.user.js&id=e3aa1af9-22e3-4b2a-831e-17c7d4f83803:1273 TypeError: Failed to fetch
fetch.then.then.catch.err @ userscript.html?name=Cosmic%20Cat.user.js&id=e3aa1af9-22e3-4b2a-831e-17c7d4f83803:1273
featured?feature=watch:1 Access to fetch at 'https://accounts.google.com/ServiceLogin?service=youtube&passive=1209600&continue=https%3A%2F%2Fwww.youtube.com%2FgetAccountSwitcherEndpoint&followup=https%3A%2F%2Fwww.youtube.com%2FgetAccountSwitcherEndpoint' (redirected from 'https://www.youtube.com/getAccountSwitcherEndpoint') from origin 'https://www.youtube.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
userscript.html?name=Cosmic%20Cat.user.js&id=e3aa1af9-22e3-4b2a-831e-17c7d4f83803:1239 [Accounts] Failed to fetch account data:
 TypeError: Failed to fetch
fetch.then.then.catch.err @ userscript.html?name=Cosmic%20Cat.user.js&id=e3aa1af9-22e3-4b2a-831e-17c7d4f83803:1239
userscript.html?name=Cosmic%20Cat.user.js&id=e3aa1af9-22e3-4b2a-831e-17c7d4f83803:1326 Object
userscript.html?name=Cosmic%20Cat.user.js&id=e3aa1af9-22e3-4b2a-831e-17c7d4f83803:1326 Object
userscript.html?name=Cosmic%20Cat.user.js&id=e3aa1af9-22e3-4b2a-831e-17c7d4f83803:3 jQuery.Deferred exception: Cannot read property 'Channels3' of undefined TypeError: Cannot read property 'Channels3' of undefined
    at HTMLDocument.<anonymous> (chrome-extension://gghkmeckbakcapojibloccjdaecmhlch/userscript.html?name=Cosmic%2520Cat.user.js&id=e3aa1af9-22e3-4b2a-831e-17c7d4f83803:2863:114)
    at Generator.next (<anonymous>)
    at fulfilled (chrome-extension://gghkmeckbakcapojibloccjdaecmhlch/userscript.html?name=Cosmic%2520Cat.user.js&id=e3aa1af9-22e3-4b2a-831e-17c7d4f83803:63:62) undefined
S.Deferred.exceptionHook @ userscript.html?name=Cosmic%20Cat.user.js&id=e3aa1af9-22e3-4b2a-831e-17c7d4f83803:3
userscript.html?name=Cosmic%20Cat.user.js&id=e3aa1af9-22e3-4b2a-831e-17c7d4f83803:3 Uncaught TypeError: Cannot read property 'Channels3' of undefined
    at HTMLDocument.<anonymous> (userscript.html?name=Cosmic%20Cat.user.js&id=e3aa1af9-22e3-4b2a-831e-17c7d4f83803:2863)
    at Generator.next (<anonymous>)
    at fulfilled (userscript.html?name=Cosmic%20Cat.user.js&id=e3aa1af9-22e3-4b2a-831e-17c7d4f83803:63)
The resource https://i.ytimg.com/generate_204 was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.
